### PR TITLE
[README] Update the README

### PR DESCRIPTION
**Pull request by stephenmcgruer**
- Update the image to not show the defunct enrollment UX
- Remove the content that is now in the explainer
- Link to the explainer and spec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,331 +4,12 @@ Using FIDO-based authentication to securely confirm payments initiated via the P
...
-
-The authenticator signature is emphemeral to each transaction and is not useful to any party unless it obtains the public key from the issuing bank.
-
-
-[PublicKeyCredential]: https://www.w3.org/TR/webauthn/#iface-pkcredential
-[Web Authentication]: https://www.w3.org/TR/webauthn/
-[pilot]: https://bit.ly/webauthn-to-pay-2020h2-pilot
+![Screenshot](payment.png)
 
 ## FAQ
```
Comment (stephenmcgruer) [inline README.md:14]: Do we want to keep this? I'm not sure.

**Comment by stephenmcgruer** [comment]
Please double-check all the stuff I removed :)

**Review by ianbjacobs — COMMENTED** [review]
Hi @stephenmcgruer,

+1 to shrinking this as you've done. I propose the following

=====

Secure Payment Confirmation (SPC) is a Web API to support streamlined authentication during a payment transaction. It is designed to scale authentication across merchants, to be used within a wide range of authentication protocols, and to produce cryptographic evidence that the user has confirmed transaction details. The [W3C Web Payments Working Group](https://www.w3.org/Payments/WG/) is developing SPC.

- [Explainer](explainer.md)
- [Specification](https://w3c.github.io/secure-payment-confirmation/) ([spec.bs](spec.bs))
- [Use cases](scope.md)
- [Requirements](requirements.md)
- Tests (coming soon!)

**Comment by stephenmcgruer** [comment]
>  I propose the following

Love it, done!

**Review by ianbjacobs — APPROVED** [review]

**Pull request merged**